libsqlite3-sys/build.rs

```diff
@@ -261,6 +261,57 @@ mod build_bundled {
                 cfg.file("sqlite3/wasm32-wasi-vfs.c");
             }
         }
+        if env::var("TARGET") == Ok("wasm32-unknown-unknown".to_string()) {
+            // Apple clang doesn't support wasm32, so use Homebrew clang by default.
```

Comment:
IMO, this block should be reverted and be replaced with documentation stating how to setup a functional cross-compiling environment on Apple's hosts. Not everyone uses Homebrew (this wouldn't work with Nix, for example) and it would result in different errors about the path not existing vs `clang` saying "unknown target".

I see it in `wasm32-unknown-unknown-openbsd-libc`'s build script as well?

Reply:
I definitely welcome any improvements to documentation or error messages. I think that it is reasonable to find and use a Homebrew installation by default if available.

Reply:
Yeah, it's pretty weird for us to automatically do this for the user, seconding the suggestion to instead document how to set things up appropriately.